lectures/arellano.md

```diff
@@ -0,0 +1,923 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Default Risk and Income Fluctuations
+
++++
+
+In addition to what's in Anaconda, this lecture will need the following libraries:
+
+```{code-cell} ipython3
+!pip install quantecon
+```
+
+## Overview
+
+This lecture computes versions of  Arellano’s  [[Are08](https://python-advanced.quantecon.org/zreferences.html#id126)]
+model of sovereign default.
+
+The model describes interactions among default risk, output, and an
+equilibrium interest rate that includes a premium for endogenous default risk.
+
+The decision maker is a government of a small open economy that borrows from
+risk-neutral foreign creditors.
+
+The foreign lenders must be compensated for default risk.
+
+The government borrows and lends abroad in order to smooth the consumption of
+its citizens.
+
+The government repays its debt only if it wants to, but declining to pay has
+adverse consequences.
+
+The interest rate on government debt adjusts in response to the
+state-dependent default probability chosen by government.
+
+The model yields outcomes that help interpret sovereign default experiences,
+including
+
+- countercyclical interest rates on sovereign debt  
+- countercyclical trade balances  
+- high volatility of consumption relative to output  
+
+
+Notably, long recessions caused by bad draws in the income process increase the government’s
+incentive to default.
+
+This can lead to
+
+- spikes in interest rates  
+- temporary losses of access to international credit markets  
+- large drops in output, consumption, and welfare  
+- large capital outflows during recessions  
+
+
+Such dynamics are consistent with experiences of many countries.
+
+Let’s start with some imports:
+
+```{code-cell} ipython3
+:hide-output: false
+
+import matplotlib.pyplot as plt
+import quantecon as qe
+import random
+
+import jax
+import jax.numpy as jnp
+```
+
+```{code-cell} ipython3
+jax.config.update('jax_enable_x64', True)
+```
+
+## Structure
+
+In this section we describe the main features of the model.
+
++++
+
+### Output, Consumption and Debt
+
+A small open economy is endowed with an exogenous stochastically fluctuating potential output
+stream $ \{y_t\} $.
+
+Potential output is realized only in periods in which the government honors its sovereign debt.
+
+The output good can be traded or consumed.
+
+The sequence $ \{y_t\} $ is described by a Markov process with stochastic density kernel
+$ p(y, y') $.
+
+Households within the country are identical and rank stochastic consumption streams according to
+
+
+<a id='equation-utility'></a>
```

Comment:
@chappiewuzefan this isn't how you should reference the equations instead you should use the in built equation references:

```
$$
 \mathbb E \sum_{t=0}^{\infty} \beta^t u(c_t) \tag{13.1}
 $$ (equation-utility)